UnityProject/Assets/Scripts/Weapons/Melee/StunBatonActivate.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
```

Comment:
I think I'd prefer it if this was merged with `StunBaton.cs` so it's all self-contained, but it's fine

Reply:
I actually did it like that but StunBaton has to be a network behaviour (because it uses a syncvar) and StunBatonActivate is an Interaction. There's probably a way to get it all in one file but I don't think it's all that important

Reply:
You can use `NB(InteractionType)Interactable`, e.g. `NBHandActivateInteractable` 😄